src/types.ts

```diff
@@ -8,20 +8,34 @@ export interface LanguageFilterSchema extends ObjectSchemaType {
   options?: LanguageFilterOptions
 }
 
-export interface Language {
-  id: string
+export type Language = {
```

Comment:
Why changed to type?

Reply:
Copied over the updated type from Internationalized Array (another reason to have a single @sanity/languages plugin)